package.json

```diff
@@ -65,6 +65,7 @@
   },
   "dependencies": {
     "@types/jest": "^27.0.2",
+    "core-js": "*",
```

Comment:
- was trying to see if I could use `setImmediate()` to leverage when the underlying React component updated, didn't work tho 😢